src/components/manage/Controlpanels/ContentTypeLayout.jsx

```diff
@@ -125,6 +125,7 @@ class ContentTypeLayout extends Component {
     this.onCancel = this.onCancel.bind(this);
     this.onSubmit = this.onSubmit.bind(this);
     this.onEnableBlocks = this.onEnableBlocks.bind(this);
+    this.onEnableBlocksBehavior = this.onEnableBlocksBehavior.bind(this);
```

Comment:
Being a little picky here, sorry :) This is old React style binding, `bind` should not be used anymore, see my other comment below. It's just a code-style issue.

Reply:
@pnicolli Ha ha. I know. I just followed the existing code :smile: Should I change all off them?!